### PR TITLE
lots of copy updates, split banner

### PR DESCRIPTION
**Pull request by philmillman**

### CODE REVIEW

File: packages/docs-site/src/content/docs/docs/get-started/what-is-dmno.mdx [hunk 1]
```diff
@@ -10,45 +9,58 @@ At the core is **DMNO Config** - the best way to deal with configuration / envir
...
-
-We hope DMNO will become the default way to deal with config within javascript/typescript - but we have no plans to stop there!
+## Who is DMNO Config for?
 
 :::note[We designed DMNO for everyone]
 From a single developer working in a monolith to a huge team wrangling a sprawling monorepo.
 :::
 
+- **Solo developers** - who want the added safety and convenience of a full schema and validation system for their config.
+- **Teams** - who want to accelerate development work by avoiding entire classes of bugs related to config, and have a unified system to manage, and share, their config across all of their services.
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/get-started/what-is-dmno.mdx:29]: remove comma after `share`?

File: packages/docs-site/src/content/docs/docs/guides/schema.mdx [hunk 2]
```diff
@@ -15,9 +9,9 @@ If you haven't already, follow our [Quickstart](/docs/get-started/quickstart) gu
 
 {/* TODO add link to example repo? */}
 
-## Project Structure
+## Project structure
 
-Using DMNO, we define a "configuration schema" for your entire project. We break that configuration up to live alongside each of the services in your code, and provide ways for them to reference each other as needed. Each service config is defined in a `config.mts` (typescript) file that lives in a `.dmno` directory.
+Using DMNO, we define a "configuration schema" for your entire project. We break that configuration up to live alongside each of the services in your code, and provide ways for them to reference each other as needed. Each service config is defined in a `config.mts` Typescript file that lives in a `.dmno` directory.
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/guides/schema.mdx:14]: looks like the official way is `TypeScript`

File: packages/docs-site/src/content/docs/docs/integrations/astro.mdx [hunk 3]
```diff
@@ -8,11 +8,11 @@ import AstroLogo from "~icons/logos/astro";
 import TabbedCode from '@/components/TabbedCode.astro';
 import BugReportLink from '@/components/BugReportLink.astro';
 
-At DMNO we _love_ Astro (this very site is built with it). That's why we're very excited to make it easier (and safer) to use environment variables in all of your Astro apps, whether it's on the client or the server!
+At DMNO we _love_ Astro. This very site is built with it! That's why we're very excited to make it easier and safer to use environment variables in all of your Astro apps, whether it's on the client or the server.
 
 ## Setup
 
-### Install the dmno astro integration
+### Install the `dmno` astro integration
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/integrations/astro.mdx:15]: I do like that we are being consistent, but unclear to me in some situations like this which to go for?

File: packages/docs-site/src/content/docs/docs/integrations/astro.mdx [hunk 4]
```diff
@@ -56,7 +56,7 @@ See our [Schema Guide](/docs/guides/schema) for the specifics of how to write yo
 
 ### Set up Typescript
 
-Astro is built with TypeScript, and so is DMNO. This means that you get full type completion and intellisense for your config items. You just need to make sure the typescript compiler knows about them
+Astro is built with TypeScript, and so is DMNO. This means that you get full type completion and Intellisense for your config items. You just need to make sure the Typescript compiler knows about them
```
Comment (theoephraim) [inline packages/docs-site/src/content/docs/docs/integrations/astro.mdx:59]: `IntelliSense` - https://code.visualstudio.com/docs/editor/intellisense

**Review by theoephraim — APPROVED** [review]

File: packages/docs-site/src/content/docs/docs/guides/schema.mdx (hunk 2 above)
Comment (philmillman) [reply packages/docs-site/src/content/docs/docs/guides/schema.mdx:14]: oops, will update

File: packages/docs-site/src/content/docs/docs/integrations/astro.mdx (hunk 3 above)
Comment (philmillman) [reply packages/docs-site/src/content/docs/docs/integrations/astro.mdx:15]: yeah I could go either way on `dmno` vs DMNO, typically `dmno` should be specific to anything involving the `dmno` package, if that makes sense?

**Pull request merged**